docs/Manage-Space.rst

```diff
@@ -58,48 +58,57 @@ Accessing a space
 =================
 
 After logging in eXo Platform successfully, you will see a list of your
-spaces under MY SPACES on the left pane of the page.
+last visited spaces with a short list on the left pane of the page, and you can see all spaces via **SPACES**.
```

Comment:
last visited spaces with a short list on the left pane of the page, and you can see all spaces via **SPACES**
```suggestion
The 5 last visited spaces are visible on the left pane, and you can see all spaces via **SPACES** in the middle pane.
```

Reply:
Actually, we're displaying at least 5, if the user is member of more than 5 spaces, and we improved this behavior and make it related to the sidebar length, so i found that it will be too much developed to be described in the documentation, as long as the spaces number to display is different from a user to another.